### ISSUE TITLE
Call to undefined method ParsedownExtended::blockMermaid()

### ISSUE THREAD

**Issue by BrainStone** [BenjaminHoegh/ParsedownExtended#16]
It appears that you forgot to add the blocks for mermaid and chart.js.

```
PHP message: PHP Fatal error:  Uncaught Error: Call to undefined method ParsedownExtended::blockMermaid() in .../includes/parsedown_extended/ParsedownExtended.php:274
Stack trace:
#0 .../includes/parsedown_extended/ParsedownExtended.php(195): ParsedownExtended->linesElements(Array)
#1 .../includes/parsedown_extended/ParsedownExtended.php(1193): ParsedownExtended->textElements('# Test\n\n## TOC ...')
#2 /var/www/dev-aura-dev/public_html/markdown.php(68): ParsedownExtended->text('# Test\n\n## TOC ...')
#3 {main}
  thrown in .../includes/parsedown_extended/ParsedownExtended.php on line 274
```

Happens with:
- Parsedown-Extra: 0.8.0-beta-1
- Parsedown: v1.8.0-beta-6

**Comment by BenjaminHoegh** [comment]
What version of Parsedown do you use?

**Comment by BrainStone** [comment]
v1.8.0-beta-6

**Comment by BenjaminHoegh** [comment]
Resolved in release [v1.0-beta-5](https://github.com/BenjaminHoegh/ParsedownExtended/releases/tag/1.0-beta-5)

**Comment by BrainStone** [comment]
Works fine. Thank you very much!
